tests/api_tests/test_participant_summary_api.py

```diff
@@ -2399,8 +2493,6 @@ def testQuery_manyParticipants(self):
             self.assertResponses("ParticipantSummary?_count=2&_sort=firstName", [[ps_1, ps_3], [ps_2]])
             self.assertResponses("ParticipantSummary?_count=2&_sort:asc=firstName", [[ps_1, ps_3], [ps_2]])
             self.assertResponses("ParticipantSummary?_count=2&_sort:desc=firstName", [[ps_2, ps_3], [ps_1]])
-            self.assertResponses("ParticipantSummary?_count=2&_sort=dateOfBirth", [[ps_2, ps_1], [ps_3]])
-            self.assertResponses("ParticipantSummary?_count=2&_sort:desc=dateOfBirth", [[ps_3, ps_1], [ps_2]])
```

Comment:
For the other assertions being removed: what do you think of changing the user's role so that the tests can continue to verify what they've been verifying? Are there other places in the tests that still verify that things like `gt` filtering work for DOB?

But for these two one in particular: why were the checks for ordering by DOB removed?

Reply:
Sure,  ill add them back with the user's role being updated.